core/templates/dev/head/domain/topic/TopicUpdateService.js

```diff
@@ -230,6 +228,61 @@ oppia.factory('TopicUpdateService', [
           // Get the current change list.
           var currentChangeList = UndoRedoService.getChangeList();
           var indicesToDelete = [];
+          var oldSubtopicIdSkillIdMap = {};
+          var index = null;
```

Comment:
Please use a better name for this; I have no idea what it represents conceptually.

Reply:
Oh, I had named that temporarily to test, will change it.

Reply:
Done, removed both of them. Not needed in the new method.